scripts/verdaccio.yaml

```diff
@@ -2,226 +2,25 @@ storage: .verdaccio-cache
 
 max_body_size: 60mb
 
-auth:
-  auth-memory:
-    users:
-      foo:
-        name: foo
-        password: s3cret
-      bar:
-        name: bar
-        password: s3cret
-
-uplinks:
-  npmjs:
-    url: https://registry.npmjs.org/
-    cache: true
+# no auth needed, this is just for e2e testing
+auth: {}
```

Comment:
I'm confused about the term "e2e testing" here.

Is auth not required to publish to verdaccio?